cli.js

```diff
@@ -71,6 +71,7 @@ var cli = meow([
 	'  --watch, -w      Re-run tests when tests and source files change',
 	'  --source, -S     Pattern to match source files so tests can be re-run (Can be repeated)',
 	'  --timeout, -T    Set global timeout',
+	'  --pool-size, -z  Set worker pool size (BETA)',
```

Comment:
I think it worked well with watcher. A limited pool of people used it for a while (mostly people who watch AVA closely and contribute), and we released it a short time later.

We could add a mention in the release notes and tweet about it to get more eyeballs on it, but I think leaving it undocumented gives us more freedom to make breaking changes. We are pre `1.0.0`, so it's not like breaking changes are a huge deal, but they are an annoyance. We have so many users now, I think it's a good idea to signal to them when they are playing on the bleeding edge (at least when we can).